apps/common/index.js

```diff
@@ -0,0 +1,69 @@
+const saveLicenceTypeSelection = require('./behaviours/save-licence-type-selection');
```

Comment:
I would suggest to rename this behaviour to something more generic, so that can be reused in future
`customSaveValues` for instance
`custom-save-values.js`